server/services/workAndSkillsPageService.test.ts

```diff
@@ -90,21 +101,55 @@ describe('WorkAndSkillsService', () => {
           todaysDate,
         )
         expect(res.learnerEducation).toEqual([
-          { key: { text: 'string' }, value: { text: 'Planned end date on 1 March 2023' } },
+          { key: { text: 'GCSE Maths' }, value: { text: 'Planned end date on 31 July 2016' } },
+          { key: { text: 'GCSE English' }, value: { text: 'Planned end date on 1 June 2022' } },
         ])
       })
 
-      it.each(['ABC123', 'DEF321'])(
```

Comment:
I'm not sure of the significance of this test iterating over several prisoner numbers (I cant see how one prisoner number should be any different to another in this test), so my new tests simply test once with one prisoner number